src/app/options/options.component.ts

```diff
@@ -35,8 +35,21 @@ export class OptionsComponent {
       });
     }),
   );
+  submitting$: Observable<boolean> = of(false);
```

Comment:
Can you add a test for when submitting$ is expected to be true and when it is false?